tls/ttls.c

```diff
@@ -1335,10 +1334,7 @@ ttls_send_alert(TlsCtx *tls, unsigned char lvl, unsigned char msg)
 	io->alert[0] = lvl;
 	io->alert[1] = msg;
 
-	if (msg == TTLS_ALERT_MSG_CLOSE_NOTIFY || lvl == TTLS_ALERT_LEVEL_FATAL)
-		close = true;
-
-	return ttls_write_record(tls, NULL, close);
+	return ttls_write_record(tls, NULL, false);
```

Comment:
This is the only place calling `ttls_write_record()`, so we don't need the boolean argument. `__ttls_send_record()` called by `ttls_write_record()` seems also don't need the last argument as well. I didn't check deeper, but please check other calls for the boolean argument.